src/cloud-cli/meltano/cloud/api/config.py

```diff
@@ -16,9 +16,10 @@
     from cached_property import cached_property
 else:
     from functools import cached_property
-MELTANO_CLOUD_BASE_URL = "https://internal.api.meltano.cloud/"
-MELTANO_CLOUD_BASE_AUTH_URL = "https://auth.meltano.cloud"
-MELTANO_CLOUD_APP_CLIENT_ID = "45rpn5ep3g4qjut8jd3s4iq872"
+
+MELTANO_CLOUD_BASE_URL = "https://internal.api-dev.meltano.cloud/"
+MELTANO_CLOUD_BASE_AUTH_URL = "https://auth-dev.meltano.cloud"
+MELTANO_CLOUD_APP_CLIENT_ID = "4dttq1k362rk7omfjchrdrt7c7"
```

Comment:
We probably don't want to default to the dev settings

Reply:
Whoops

Reply:
Fixed in 6002a09. Thanks!